src/devSVG.cpp

```diff
@@ -429,10 +495,25 @@ void svg_path(double *x, double *y,
 double svg_strwidth(const char *str, const pGEcontext gc, pDevDesc dd) {
   SVGDesc *svgd = (SVGDesc*) dd->deviceSpecific;
 
-  gdtools::context_set_font(svgd->cc, fontname(gc->fontfamily, gc->fontface),
-    gc->cex * gc->ps, is_bold(gc->fontface), is_italic(gc->fontface));
+  std::string file = fontfile(gc->fontfamily, gc->fontface, svgd->font_aliases, svgd->font_extra);
+  gdtools::context_set_font(svgd->cc, fontname(gc->fontfamily, gc->fontface, svgd->font_aliases),
+                            gc->cex * gc->ps, is_bold(gc->fontface), is_italic(gc->fontface), file);
   FontMetric fm = gdtools::context_extents(svgd->cc, std::string(str));
 
+  if (fm.width == 0) {
```

Comment:
This is starting to get esoteric, but what if the character actually has zero width? i.e. a zero-width space.  I think maybe, `fm` needs to a have a success flag?  But we can ignore for now.